sample/src/com/vungle/publisherSampleAndroid/MainActivity.java

```diff
@@ -170,4 +202,11 @@ protected void onResume() {
 		super.onResume();
 		vunglePub.onResume();
 	}
+
+	@Override
+	protected void onDestroy() {
+		// onDestroy(), remove eventlistners.
```

Comment:
typo in `listner`, probably don't need this comment unless we're letting devs know why we're removing these listeners (i.e. to to free up the listener on deallocation, since the original activity is no longer in scope it won't need to handle SDK events anymore)